.github/workflows/ci.yml

```
@@ -29,18 +110,24 @@ jobs:
-      - name: Run tests
-        run: npm test -- --run
+      - name: Run tests with coverage
+        run: npm test -- --run --coverage
+
+      - name: Check test results
+        run: |
+          echo "✅ All tests passed on Node ${{ matrix.node }} (${{ matrix.os }})"
 
       - name: Upload coverage
         if: matrix.node == 20 && matrix.os == 'ubuntu-latest'
         uses: codecov/codecov-action@v3
```

actionlint (1.7.8)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)